pdc_client/plugins/auth_group_resource_permissions.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+
+from __future__ import print_function
+
+import json
+
+from pdc_client.plugin_helpers import (PDCClientPlugin,
+                                       extract_arguments,
+                                       add_parser_arguments,
+                                       add_create_update_args)
+
+
+class AuthGroupResourcePermissionPlugin(PDCClientPlugin):
+    def register(self):
+        self.set_command('auth/group-resource-permissions')
+        list_parser = self.add_action('list', help='list all groups and their resource permission')
+        add_parser_arguments(list_parser, {'resource': {},
+                                           'permission': {},
+                                           'group': {}},
+                             group='Filtering')
+        list_parser.set_defaults(func=self.list_group_resource_permission)
+
+        info_parser = self.add_action('info', help='display details about a group resource permission')
+        info_parser.add_argument('id', metavar='ID')
+        info_parser.set_defaults(func=self.group_resource_permission_info)
+
+        update_parser = self.add_action('update', help='update an existing group resource permission')
+        update_parser.add_argument('id', metavar='ID')
+        self.add_create_update_argument(update_parser, required=True)
+        update_parser.set_defaults(func=self.group_resource_permission_update)
+
+        create_parser = self.add_action('create', help='create new group resource permission')
```

Comment:
For user, is it better to let help text to be something like `grant a resource permission to a group`. `create` is what dev implemented. We don't have describe it from developer's point of view to our customer. 

Same as update, but I am not sure in update how to make it clear and friendly.